config/initializers/money.rb

```diff
@@ -40,9 +39,23 @@
                              type: :string,
                              present: true,
                              null: false,
-                             default: 'USD'
+                             default: 'USD_6'
```

Comment:
I would use 8 to 12 digits instead of 6 (what I've seen in the past)
I would suggest you define a conversion to dollars, and then you print dollars.